spyder_kernels/utils/nsview.py

```diff
@@ -328,16 +328,16 @@ def collections_display(value, level):
 def value_to_display(value, minmax=False, level=0):
     """Convert value for display purpose"""
     # To save current Numpy threshold
```

Comment:
Please change the comment above to say `printoptions` instead of `threshold` @juanis2112